app/helpers/materialize_view_helper.rb

```diff
@@ -6,5 +6,11 @@ def refresh_materialized_view(concurrently: false)
     GeographicalArea.refresh!(concurrently:)
     GeographicalAreaDescription.refresh!(concurrently:)
     GeographicalAreaDescriptionPeriod.refresh!(concurrently:)
+    AdditionalCode.refresh!(concurrently:)
```

Comment:
We could just register these when the configure method of the oplog plugin is called and then iterate over them